lib/app/ui/grid/grid_page.dart

```diff
@@ -168,30 +174,38 @@ class _GridScreenState extends State<GridScreen> {
   }
 
   Widget drawDriversAndSummary() {

```

Comment:
Ez a függvény túlságosan hosszú és már a neve is sugallja, hogy egynél több dolgot csinál. Ez sérti a SOLID elvek közül a Single Responsibility Principle-t

Ezt a függvényt ketté kéne szedni:
- legyen egy olyan, ami a versenyzőket rajzolja ki
- legyen egy olyan, ami a summary-t rajzolja ki
- ehhez a két dologhoz szükséges adat ne a UI osztályon és ne is a ViewModel-en álljon elő, hanem a Service rétegben. Azaz, ennek a képernyőnek mindenképp legyen egy `GridService` osztálya, ahol a keresés és a summary kirajzolásához szükséges adat előáll

Code smell
> Hosszú függvény: túl nagyra nőtt funkció